File: CodeTest/Python/SWEA/D1_2070.py
testCase = int(input())

for i in range(1, testCase+1):
    num1, num2 = map(int, input().split())
    sign = ''

    if num1 > num2:
        sign = '>'
    elif num1 < num2:
        sign = '<'
    else:
        sign = '='
    
    print(f'#{i} {sign}')

File: CodeTest/Java/SWEA/D1_2070.java
import java.util.*;

public class D1_2070 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int testCase = sc.nextInt();
        int num1;
        int num2;
        String sign = null;

        for(int i=1; i<=testCase; i++) {
            num1 = sc.nextInt();
            num2 = sc.nextInt();

            if(num1 > num2) {
                sign = ">";
            } else if(num1 < num2) {
                sign = "<";
            } else {
                sign = "=";
            }

            System.out.println("#" + i + " " + sign);            
        }
    }
}